crates/rolldown_testing_config/src/lib.rs

```diff
@@ -42,6 +42,9 @@ pub struct TestMeta {
   /// If `true`, the bundle will be called with `write()` instead of `generate()`.
   #[serde(default = "true_by_default")]
   pub write_to_disk: bool,
+  /// If `true`, assets won't be snapshot
+  #[serde(default)]
+  pub snapshot_skip_assets: bool,
```

Comment:
Added because without this option, the snapshot will be
```
---
source: crates/rolldown_testing/src/integration_test.rs
snapshot_kind: text
---
# Assets

## entry.js

```js

//#region \0virtual:batman
var __virtual_batman_default = "na na na na na";

//#endregion
//#region \0virtual:/files/projects/rolldown/crates/rolldown_plugin_virtual/tests/src/robin.js  😨😨😨😨
var robin_default = "batman";

//#endregion
//#region src/entry.js
console.assert(__virtual_batman_default === "na na na na na");
console.assert(robin_default === "batman");

//#endregion
```
the path is not stable across machines. If there's a better way, please let me know.